### DISCUSSION TITLE
[UI Design idea] Combine station logbook and locations into one page?

### DISCUSSION THREAD

**Discussion by ei8fdb** [magicbug/Cloudlog#3220] [Show and tell]
I've started writing some documentation for station logbooks and locations. As I've been reviewing what a new user sees, I thought the following:

Right now Cloudlog gives a new user prompting to create a new logbook and location. This is helpful.

To do this they've got to navigate to 2 seperate pages.

If they create a logbook but not a location (vice-versa) they still see the prompt to create the station location. This is also helpful.

To make the relationship between Station logbook and location clearer, and save some unnecessary clicking - would it be possible to combine those 2 pages?

The empty state (i.e. when nothing is created) and the created state, it looks OK, it doesn't look to cluttered - see wireframe below.

![Image shows UI design idea combining Cloudlog Station logbook UI and Station Locations into one page](https://github.com/user-attachments/assets/7666a9b8-e267-4089-89c3-0424ad1f290b)

There might be a benefit to users - new and experienced. Is there any benefit from programming point of view? Less to maintain? Easier to change?

Any thoughts?

**Comment by magicbug** [comment]
I have considered having it on one page, but I've been wary of implementing it that being said the work flow of creating a logbook needs improving. I'm not sure in general the best way forward for these to make a more unified feel
